source/bsondump/bsondump-examples.txt

```diff
@@ -0,0 +1,47 @@
+.. _bsondump-examples:
+
+=================
+bsondump Examples
+=================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. |arrow| unicode:: U+27A4
+.. |tool-binary| replace:: ``bsondump``
+
+.. program:: bsondump
+
+This page shows examples for |tool-binary|.
+
+.. include:: /includes/extracts/require-cmd-line-bsondump.rst
```

Comment:
I think this include contains the sentence "Run [bsondump] from the system command line, not the 
[mongo](https://www.mongodb.com/docs/manual/reference/mongo/#mongodb-binary-bin.mongo)
 shell."
Since the mongo shell is deprecated, could we update the include to point to mongosh instead?